catalog-rest-service/src/main/resources/json/schema/entity/services/connections/metadata/metadataESConnection.json

```diff
@@ -9,15 +9,15 @@
     "metadataESType": {
       "description": "Metadata to Elastic Search type",
       "type": "string",
-      "enum": ["MetadataES"],
-      "default": "MetadataES"
+      "enum": ["MetadataElasticsearch"],
+      "default": "MetadataElasticsearch"
```

Comment:
Sorry curiosity, Why did we need to change this name?

Reply:
So [here](https://github.com/open-metadata/OpenMetadata/blob/6701ae64163db812f1c43f1cf924e8550130f90d/ingestion/src/metadata/utils/class_helper.py#L23) we are checking if the formatted  `source.type` matches any value in service enums. in case of metadata_to_es pipeline we are using the `source.type = metadata_elasticsearch` and we are using the `serviceConnection.config.type = MetadataES` due to which at the comparison logic we were comparing `"metadataelasticsearch" == "metadataes"`.

To solve this problem I renamed the enum to `MetadataElasticsearch` so after applying the formatting logic we are able to fetch the correct service type. also this kind of standardizes how we name the service type. for example in mysql or any other source we have `source.type = mysql` and `serviceConnection.config.type = Mysql`, `MetadataES` was the only odd man out. 


Let me know wdyt, if you have any better solution for this problem.

Reply:
got it thanks for the explanation